setup.py

```diff
@@ -39,7 +39,7 @@
       author_email='info@precice.org',
       license='LGPL-3.0',
       packages=['fenicsxprecice'],
-      install_requires=['pyprecice~=2.0.0', 'scipy', 'numpy>=1.13.3', 'mpi4py'],
+      install_requires=['pyprecice~=3.0.0', 'scipy<1.15.0', 'numpy>=1.13.3', 'mpi4py'],
```

Comment:
```suggestion
      install_requires=['pyprecice~=3.0', 'scipy<1.15.0', 'numpy>=1.13.3', 'mpi4py'],
```
Using `pyprecice~=3.0.0` would only allow `3.0` releases and not `3.1` etc. See https://stackoverflow.com/questions/39590187/in-requirements-txt-what-does-tilde-equals-mean.